docs/schema/reference.md

```diff
@@ -222,7 +222,7 @@ The [Bid statistics and details](https://extensions.open-contracting.org/en/exte
 
 ### Award
 
-The award section is used to announce any awards issued for this tender. There can be multiple awards made. Releases can contain all, or a subset, of these awards. A related award block is required for every contract block, as the award contains information on the suppliers. In particular cases there can be multiple suppliers for a single award: for example, in the case of [consortia](../guidance/map/buyers_suppliers.md#consortia-suppliers) and in [framework agreements](../guidance/map/framework_agreements).
+The `awards` array is used to announce any awards issued for this tender. There can be multiple awards made, each of which must be detailed using the `Award` subschema. Releases can contain all, or a subset, of these awards. Awards contain information about suppliers. In particular cases there can be multiple suppliers for a single award: for example, in the case of [consortia](../guidance/map/buyers_suppliers.md#consortia-suppliers) and in [framework agreements](../guidance/map/framework_agreements).
```

Comment:
Remove contradictory sentence.

```suggestion
The `awards` array is used to announce any awards issued for this tender. Releases can contain all, or a subset, of these awards. Awards contain information about suppliers. In particular cases there can be multiple suppliers for a single award: for example, in the case of [consortia](../guidance/map/buyers_suppliers.md#consortia-suppliers) and in [framework agreements](../guidance/map/framework_agreements).
```

Reply:
I don't see how this is contradictory? It's trying to make it clear that the `awards` array (of which there is only one) is made up of `Award` objects (of which there can be multiple). This is why the word awards in this sentence isn't surrounded by backticks, because it's just referring to the concept, not the object. Without this I don't think it's clear how the `Award` object is related to the `awards` array, or how the `awards` array is populated.

Reply:
I read the sentence as setting a normative rule that all awards (in the real-life contracting process) must appear in `awards`, whilst the following sentence says that a release can contain only a subset of the awards.

I don't think we need to say "there can be multiple", otherwise we'd need to explain that for each array in the schema.

I'm not sure it's important to mention the subschema, I think it would be sufficient to add an introductory sentence like "Each `Award` has the following fields:" before the jsonschema directive.

Reply:
@jpmckinney this comes up in a few places so it would be good to know what you think - do we need to explicitly mention the subschema used in arrays like `awards`, `contracts`, `amendments` etc.?

Reply:
I remember some of my thinking about this as I did give this a lot of thought at the time. This is linked to the consideration of entirely restructuring this page https://github.com/open-contracting/standard/issues/1075 The subheadings under "Release structure" are all supposed to be the high level objects. So 'planning', 'tender' etc. But while `planning', 'tender' are single objects, `awards` and `contracts` are arrays of objects. It's the objects within the array that are being described, but the heading is talking about the array. So it needs to be made clear that the fields being discussed aren't in the array, they're in the objects that the array contains. Ideally I think `Award` and `Contract` need their own sub-sections in the "Subschema reference" section similar to how `Document` and `Period` are in there. I suspect the thinking behind not putting them there is that `Award` and `Contract` can only be used in a single place each, but they can both be used multiple times which I think makes them subschema and therefore deserving of their own subsection in the "Subschema reference" section. That would leave the `awards` and `contracts` subsections under "Release structure" to not have to mention them. There's already a lot of confusion between which subschema are in the "Subschema reference" section and which are under "Release structure" subsections, e.g. `Parties` and `Organization`, but that's already being discussed (or at least collected) in

Reply:
Yeah, I'd also favour listing `Award` and `Contract` in the subschema section for consistency. I think that'll be addressed as a part of the larger restructure discussed in https://github.com/open-contracting/standard/issues/1075.

> It's the objects within the array that are being described, but the heading is talking about the array. So it needs to be made clear that the fields being discussed aren't in the array, they're in the objects that the array contains.

Hmm, fields (key-value pairs) can't be 'in an array' without belonging to an object so I'm not sure this really needs explaining, but if you think it's important to be more explicit about the relationship between the `awards` array and `Award` objects, perhaps the sentence introducing the jsonschema directive could be:

> The items in the `awards` array are `Award` objects. Each `Award` has the following fields:

Reply:
I'm happy with that suggestion, I'll make that change and do the same for the other instances of the same issue

Reply:
> @jpmckinney this comes up in a few places so it would be good to know what you think - do we need to explicitly mention the subschema used in arrays like awards, contracts, amendments etc.?

Without the context of the full PR, I would say such sentences are unnecessary. We generally have a subschema for each array, so it would be kind of repetitive.